src/components/ChooseTickets.vue

```diff
@@ -102,6 +111,20 @@ export default {
         })),
       };
     },
+    bookingData() {
+      const tickets = [];
+      this.ticketsData.forEach((ticket) => {
+        tickets.push({
+          title: this.selectedMovie.movie.title,
+          seat: `Row ${ticket.seat[0]}, Seat ${ticket.seat.slice(1)}`,
+          datetime: `${getDayOfWeek(this.selectedMovie.datetime)}
+           ${getDate(this.selectedMovie.datetime)} -
+           ${getTime(this.selectedMovie.datetime)}`,
+          ticketType: ticket.ticket.name,
+        });
+      });
+      return tickets;
```

Comment:
You could use `map` here instead and not need to make a temporary array 💡
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map